core/src/main/scala/tapir/Schema.scala

```diff
@@ -37,5 +39,18 @@ object Schema {
     def show: String = s"ref($fullName)"
   }
 
+  case class SCoproduct(schemas: List[Schema], discriminator: Option[Discriminator[_]]) extends Schema {
+    override def show: String = "allOf:" + schemas.mkString(",")
+  }
+
   case class SObjectInfo(shortName: String, fullName: String)
+
+  case class Discriminator[T](propertyName: String, mapping: Map[String, Schema])
```

Comment:
what's `mapping` in here`?

Reply:
ah ok, I re-read https://swagger.io/specification/#discriminatorObject :)

so mapping is a mapping from property values to the schemas which correspond to those values, right?

Reply:
reading on, the `mapping` is actually an override. By default, the value of the the property points to the name of the *object schema* to use. This can be oveeriden by an entry in `mappings` which points to an alternate schema - so maybe `mapping` should be `mappingOverride: Map[String, SchemaRef]`?

Reply:
+1 for `mappingOverride`
` Map[String, SchemaRef]` - how would you create such mapping?